torchchat/utils/scripts/install_et.sh

```diff
@@ -19,10 +19,4 @@ pushd ${TORCHCHAT_ROOT}
 find_cmake_prefix_path
 clone_executorch
 install_executorch_libs $ENABLE_ET_PYBIND
-install_executorch_python_libs $ENABLE_ET_PYBIND
```

Comment:
Move this to install_exeuctorch_python_libs because on the non-cache path in the CI, having it here doesn't work.

Reply:
If we remove this line: `install_executorch_python_libs $ENABLE_ET_PYBIND`

Where is this being called?

Reply:
It is called inside install_executorch_libs (which installs both c++ and python), so it was redundant